app.go

```diff
@@ -770,7 +770,7 @@ func (app *App) provide(p provide) {
 
 		switch {
 		case p.IsSupply:
-			app.log.LogEvent(&fxevent.Supply{TypeName: p.SupplyType.String()})
+			app.log.LogEvent(&fxevent.Supplied{TypeName: p.SupplyType.String()})
```

Comment:
Both, Provide and Supply events are emitted after the operation. (This is a `defer` block.)